applications/ChimeraApplication/custom_strategies/strategies/fs_strategy_for_chimera.h

```diff
@@ -168,23 +168,6 @@ class FSStrategyForChimera : public FSStrategy<TSparseSpace,TDenseSpace,TLinearS
     ///@{
 
 
-   void SetActiveStateOnConstraint(const Flags& TheFlagToSet ,const bool ValToSet)
```

Comment:
so this can be removed altogether?
Nice

Reply:
Yes since they are on different sub modelparts, and those sub modelparts are used in the respective vel and pressure builder and solvers, this is no longer necessary.